Apps/SampleData/models/BoxInstanced/BoxInstanced.gltf

```diff
@@ -95,9 +95,7 @@
         "EXT_instance_features": {
           "featureIds": [
             {
-              "propertyTable": 0,
-              "offset": 0,
-              "repeat": 1
+              "propertyTable": 0
```

Comment:
`TriangleFaceBox.gltf` and `CustomShaderGuide/README.md` still have `"offset": 0` as well